Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -424,10 +424,12 @@
   description: The jumpsuit for the poor sop with a mop.
   components:
   - type: Sprite
-    sprite: Clothing/Uniforms/Jumpsuit/janitor.rsi
+#    sprite: Clothing/Uniforms/Jumpsuit/janitor.rsi # Commented out on Harmony
+    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/janitor.rsi # Updated sprite pathing for Harmony
   - type: Clothing
-    sprite: Clothing/Uniforms/Jumpsuit/janitor.rsi
-
+#    sprite: Clothing/Uniforms/Jumpsuit/janitor.rsi # Commented out on Harmony
+    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/janitor.rsi # Updated sprite pathing for Harmony
```

This can be simplified
```suggestion
    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/janitor.rsi # Updated sprite pathing for Harmony
```